dffml/service/dev.py

```diff
@@ -1056,6 +1060,96 @@ class Bump(CMD):
     packages = BumpPackages
 
 
+class SphinxBuildError(Exception):
+    """
+    Raised when sphinx-build command exits with a non-zero error code.
+    """
+
+    def __str__(self):
+        return "[ERROR] Failed run sphinx, is it installed (pip install -U .[dev])?"
+
+
+class MakeDocs(CMD):
+    """
+    Generate docs of the complete package 
+    """
+
+    async def run(self):
+
+        root = Path(__file__).parents[2]
+
+        pages_path = root / "pages"
+
+        shutil.rmtree(pages_path, ignore_errors=True)
+
+        docs_path = root / "docs"
+
+        files_to_check = [
+            ("changelog.md", "CHANGELOG.md"),
+            ("shouldi.md", "examples/shouldi/README.md"),
+            ("swportal.rst", "examples/swportal/README.rst"),
+            (
+                "contributing/consoletest.md",
+                "dffml/util/testing/consoletest/README.md",
+            ),
+        ]
+
+        for symlink, source in files_to_check:
+            file_path = docs_path / symlink
+            if not file_path.exists():
+                file_path.symlink_to(root / source)
+
+        # HTTP Service Docs
+        service_path = docs_path / "plugins/service"
+        service_path.mkdir(parents=True, exist_ok=True)
+        http_docs_pth = service_path / "http"
+        os.unlink(http_docs_pth)
```

Comment:
The bash script was just unlinking every time to avoid an `if`. Let's use `pathlib.Path.unlink` instead of the `os` function, for consistency, and just pass `missing_ok=True)`

Reply:
`missing_ok=True` is only available in `python3.8` and above. That's why I'd use a if statement to avoid `FileNotFoundError`